src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -752,7 +755,19 @@ def handle_after_hours_message(
         db_session=db_session, incident_id=context["subject"].id, email=user.email
     )
     # get their timezone from slack
-    owner_tz = (dispatch_slack_service.get_user_info_by_email(client, email=owner_email))["tz"]
+    try:
+        owner_tz = (dispatch_slack_service.get_user_info_by_email(client, email=owner_email))["tz"]
+    except SlackApiError as e:
+        if e.response["error"] == "users_not_found":
+            e.add_note(
+                "This error usually indiciates that the incident commanders Slack account is deactivated."
+            )
+
+        log.warning(f"Failed to fetch timezone from Slack API: {e}")
+        owner_tz = (
+            "UTC"  # set a default timezone value (change this to your preferred default timezone)
+        )
```

Comment:
I'm going to cut a PR to monitor and deactivate incident participants whose employment status have changed to avoid all these `users_not_found` errors. If the participant role has the incident commander or another important role such as Scribe or Liaison, we can alert in the conversation, so whoever is there can make the required changes.

Reply:
Sounds good, that's a lot more robust. Do you want to merge this until we have that implemented or should we close this out and wait for the better solve?

Reply:
We can merge it and rely on it until we have the more robust approach in place.